### DISCUSSION TITLE
[Day 11 - 5/10] TripPy & CHAN-DST

### DISCUSSION THREAD

**Discussion by karim-moon** [bcaitech1/p3-dst-chatting-day#49] [Peer Session]
### 의견

문종

- 베이스라인코드에서 train에는 KoElectra 토크나이저, inference에는 BERT토크나이저 씀
→ 바꿔도 문제는 없었다
- solution 코드는 bert-small-tokenizer, BERT 인코더 사용
- word dropout 적용안돼있음

유진

- CSV 제출하고 나서 어떤 종류의 슬롯을 잘 예측 못하는지 살펴보자

### 할거

1. baseline 업데이트 solution + word dropout 적용 + wandb 추가
2. CSV 제출하고 나서 JGA (턴에서 어떤 슬롯이 빠져서 안됐는지) 비교 + 전체 slot 에서 어떤 slot 을 잘 예측 못하는지 체크

### TripPy

- a_t_inform 이 이전턴까지 inform 한걸 알려주는데, 어떻게 현재 턴에 gate 결과가 inform 이 나올 수 있을까?

### 유진님 CHAN-DST 설명

[https://github.com/bcaitech1/p3-dst-chatting-day/discussions/47](https://github.com/bcaitech1/p3-dst-chatting-day/discussions/47)

### 역할 분담

- 희석, 창우 : SOM-DST 구현
- 문종 : 베이스라인 옮기기
- 유진, (문종) : CHAN-DST 구현
- 재우 : Transformer-DST 조사

### Data Visuallization

- **Visual Data Exploartion** 을 통해 user interaction 을 이끌어낼 수 있으므로 중요한 개념
- Visual Analytics + Deep Learning ⇒ 학습 과정을 시각적으로 보여줌으로써 디버깅에 용이함을 줌, 딥러닝은 차원이 많기 때문에 시각적 표현이 많은 도움이 됨
- VATLD, 2020년 베스트 논문

**Comment by Heeseok-Jeong** [comment]
앗 감사합니다..!
